File: OPENAPI-REST-API/openapi-client/scala-gatling/src/gatling/scala/org/openapitools/client/model/GetAccountLocales200ResponseValue.scala

package org.openapitools.client.model


case class GetAccountLocales200ResponseValue (
    /* English display name of the locale. */
    _name: Option[String],
    /* Display name of the locale in its own language. */
    _local_name: Option[String]
)
object GetAccountLocales200ResponseValue {
    def toStringBody(var_name: Object, var_local_name: Object) =
        s"""
        | {
        | "name":$var_name,"local_name":$var_local_name
        | }
        """.stripMargin
}

File: OPENAPI-REST-API/openapi-client/zapier/models/getAccountLocales_200_response_value.js
const utils = require('../utils/utils');

module.exports = {
    fields: (prefix = '', isInput = true, isArrayChild = false) => {
        const {keyPrefix, labelPrefix} = utils.buildKeyAndLabel(prefix, isInput, isArrayChild)
        return [
            {
                key: `${keyPrefix}name`,
                label: `English display name of the locale. - [${labelPrefix}name]`,
                type: 'string',
            },
            {
                key: `${keyPrefix}local_name`,
                label: `Display name of the locale in its own language. - [${labelPrefix}local_name]`,
                type: 'string',
            },
        ]
    },
    mapping: (bundle, prefix = '') => {
        const {keyPrefix} = utils.buildKeyAndLabel(prefix)
        return {
            'name': bundle.inputData?.[`${keyPrefix}name`],
            'local_name': bundle.inputData?.[`${keyPrefix}local_name`],
        }
    },
}
